cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+- name: 'mirror.gcr.io/library/golang'
+  entrypoint: make
+  args: ['format']
+- name: 'mirror.gcr.io/library/golang'
+  entrypoint: make
+  args: ['build']
+- name: 'mirror.gcr.io/library/golang'
+  entrypoint: make
+  args: ['test']
+- name: docker
+  args: [
+    'build',
+    '-t', 'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${TAG_NAME}${BRANCH_NAME}',
+    '-t', 'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${TAG_NAME}${BRANCH_NAME}-${SHORT_SHA}',
+    '-t', 'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${SHORT_SHA}',
+    '.'
+  ]
+images: [
+  'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${TAG_NAME}${BRANCH_NAME}',
+  'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${TAG_NAME}${BRANCH_NAME}-${SHORT_SHA}',
+  'gcr.io/$PROJECT_ID/stackdriver-prometheus-sidecar:${SHORT_SHA}'
+]
+options:
+ diskSizeGb: 200
```

Comment:
Nit: Indent two spaces this line and the next.

Reply:
Fixed, thanks.